docs: release @excalidraw/excalidraw 0.7.0 🎉 (#3497)
excalidraw/excalidraw: src/packages/excalidraw/CHANGELOG.md

Please add the latest change on the top under the correct section.
-->

## 0.7.0 (2021-04-26)

## Excalidraw API

### Features

> - Support scrolling to center to a single Excalidraw element, or not supplying any argument at all which will default to current elements on the scene using [`scrollToContent`](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L265) [#3482](https://github.com/excalidraw/excalidraw/pull/3482).

Reviewer: ```suggestion
- [`scrollToContent`](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L265) API now supports passing just a single Excalidraw element, or none at all (which will default to current elements on the scene) [#3482](https://github.com/excalidraw/excalidraw/pull/3482).
```

The previous change made it look like the `scrollToContent` API related to the "not supplying any argument" sentence.

- Add `onPaste` prop to handle custom clipboard behaviours [#3420](https://github.com/excalidraw/excalidraw/pull/3420).

### Fixes

- Changing z-index of elements (sorting them below/above other elements) now updates their `version` and `versionNonce` (only applies to the selected elements). This fix will allow you account for z-index changes if you're syncing just the elements that changed (and not the whole scene) [#3483](https://github.com/excalidraw/excalidraw/pull/3483).
- Only handle cut/paste events inside excalidraw [#3484](https://github.com/excalidraw/excalidraw/pull/3484).

- Make history local to a given Excalidraw instance. This fixes a case where history was getting shared when you have multiple Excalidraw components on the same page [#3481](https://github.com/excalidraw/excalidraw/pull/3481).
- Use active Excalidraw component when editing text. This fixes a case where text editing was not working when you have multiple Excalidraw components on the same page [#3478](https://github.com/excalidraw/excalidraw/pull/3478).

> - Library onClick paste off-center [#3462](https://github.com/excalidraw/excalidraw/pull/3462).

Reviewer: ```suggestion
- Fix library being pasted off-center [#3462](https://github.com/excalidraw/excalidraw/pull/3462).
```

### Features

> - Support tab in text Wyswig [#3411](https://github.com/excalidraw/excalidraw/pull/3411).

Reviewer: ```suggestion
- Support tab in text Wysiwyg [#3411](https://github.com/excalidraw/excalidraw/pull/3411).
```

- App now breaks into mobile view using the component dimensions, not viewport dimensions. This fixes a case where the app would break sooner than necessary when the component's size is smaller than viewport [#3414](https://github.com/excalidraw/excalidraw/pull/3414).

- Add screenshots to manifest.json [#3369](https://github.com/excalidraw/excalidraw/pull/3369).

- Enable drop event on the whole component [#3406](https://github.com/excalidraw/excalidraw/pull/3406).

### Fixes

- Focus on last active element when dialog closes [#3447](https://github.com/excalidraw/excalidraw/pull/3447).

> - Incorrectly caching png file handle [#3407](https://github.com/excalidraw/excalidraw/pull/3407).

Reviewer: ```suggestion
- Fix incorrectly caching png file handle [#3407](https://github.com/excalidraw/excalidraw/pull/3407).
```

> - Popover position incorrect on Safari for non-zero offsets/scroll [#3399](https://github.com/excalidraw/excalidraw/pull/3399).

Reviewer: ```suggestion
- Fix popover position incorrect on Safari for non-zero offsets/scroll [#3399](https://github.com/excalidraw/excalidraw/pull/3399).
```
